decent-dao/fractal-interface
# Remove infura rpc api key

Removes yet another API key in an environment variable!

These days it's possible to use wagmi's public client as an EIP 1193 provider within the `evm-proxy-detection` library.

Testing

1. Open up a DAO
2. Go to the Template Creation page, start a new template
3. Paste in a proxy address into the target address field
4. Notice how the underlying implementation's functions show up in the function selector!
